appveyor.yml

```diff
@@ -34,12 +34,12 @@ test_script:
   - npm run eslint
   - npm run test-cov
 
-after_script:
-  - npm install coveralls
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
-
+# after_test:
```

Comment:
I think we should remove the lines altogether, hexo doesn't have them either.

Reply:
Done